packages/faustwp-cli/utils/generatePossibleTypes.ts

```diff
@@ -1,9 +1,11 @@
-require('isomorphic-fetch');
-const fs = require('fs')
+import 'isomorphic-fetch';
+import fs from 'fs';
 
-const GRAPHQL_ENDPOINT = `${process.env.NEXT_PUBLIC_WORDPRESS_URL}/graphql`;
+import { infoLog, errorLog } from './log.js';
+
+export async function generatePossibleTypes() {
+  const GRAPHQL_ENDPOINT = `${process.env.NEXT_PUBLIC_WORDPRESS_URL}/graphql`;
```

Comment:
We have a filter for the GraphQL endpoint in `@faustwp/core` via the plugin system, but not in `@faustwp/cli`. We may need to give some additional thought on how we pull in the GraphQL endpoint in both packages.

Reply:
I appreciate the feedback!

I'm wondering if there is a need for `@faustwp/cli` to be separate from `@faustwp/core`? The separation of concerns within the monorepo makes sense, but I'm wondering if we should have the CLI bundled with core when published to npm.

This would allow us to drop
- @faustwp/core
- @faustwp/cli

and maybe pivot to `@wpengine/faust` or something similar.